jimbo1qaz/0CC-FamiTracker
# Add N163 file-specific mixing level offset.

Test procedure:
- save
- open
----
- move slider
- edit text box
- ctrl-A textbox
- File should be marked as modified.
----
- (test open corrupted file)
